src/editor/scheduler-editor-time.ts

```diff
@@ -93,6 +93,12 @@ export class SchedulerEditorTime extends LitElement {
 
   render() {
     if (!this.hass || !this.config || !this.entities || !this.actions) return html``;
+    let timePickerHeader = '';
+    if (!this.timeslots) {
+      timePickerHeader = this.hass.localize('ui.dialogs.helper_settings.input_datetime.time');
+      if (this.entities?.[0].id.startsWith('time'))
+        timePickerHeader += ` (${localize('ui.panel.common.title', getLocale(this.hass))})`;
+    }

```

Comment:
This `timePickerHeader` logic changes the section title from "TIME" to "TIME (SCHEDULER)" to help disambiguate between the two time sections — see red arrows in the screenshots below. ***This only applies when the selected entity is a time entity.*** If the user selected a different entity, say switch, the section title remains as before, i.e. simply "TIME".

_Without the `timePickerHeader` logic:_
<img width="70%" alt="New Schedule - Time Editor - ambiguous highlight" src="https://github.com/nielsfaber/scheduler-card/assets/15091591/62c6054f-650f-4128-952b-209730b58775" />

_With the `timePickerHeader` logic:_
<img width="70%" alt="New Schedule - Time Editor - highlight" src="https://github.com/nielsfaber/scheduler-card/assets/15091591/d852f07a-063c-466a-aabd-9341d617e4c8" />